src/plugins/hetero/executable_network.cpp

```diff
@@ -515,7 +515,9 @@ HeteroExecutableNetwork::HeteroExecutableNetwork(const InferenceEngine::CNNNetwo
         network._network = _heteroPlugin->GetCore()->LoadNetwork(network._clonedNetwork,
                                                                  network._device,
                                                                  metaDevices[network._device]);
+        _exeDevices += network._device + ",";
     }
+    _exeDevices.pop_back();
```

Comment:
@ilya-lavrenov for hetero, the exedevice, shall we list all the targets for sub networks, like GPU,CPU,GPU,CPU? or we only keep one occurrence of the target hw?

Reply:
@songbell @ilya-lavrenov I think we should avoid duplicates and keep execution device one time. Also maybe there is a sense to keep order according to priority.

Reply:
agree with @nkogteva 
also, why do we need to keep it as a field of class? I suppose we can create it on demand once user requested it.
Secondly, why `ov::execution_devices` is a string? does it make sense to have it to be `std::vector<std::string>`?

Reply:
changed the implementation,  please review again @ilya-lavrenov  @nkogteva